### DISCUSSION TITLE
Error and warning lines don't wrap

### DISCUSSION THREAD

**Discussion by ayushkhanduri** [w0rp/ale#4426] [Q&A - Ask for help with problems]
I am having issues reading errors and warnings as the text for error and warning isn't wrapping. I have set wrap in my vimrc which is working when my text overflows but it doesn't work with linting . Below is a screenshot on how messages overflow the screen and there is no way to read them

![Screenshot from 2023-01-21 16-39-39](https://user-images.githubusercontent.com/13096346/213864257-89d279b4-7139-4454-9e08-29fab9567d9c.png)

**Comment by w0rp** [comment]
I think virtualtext in Vim can't be word wrapped, as it would cause characters to move up and down the screen, unless it wrapped inside of the space to the right margin it might be able to fit in. For long errors on long lines, the best way to look at an error is to use the other options to show errors. You can always use `:ALEDetail` for a problem at the cursor location.
